docs/changelog.md

```diff
@@ -144,17 +144,17 @@ lint:
 
 ### Fixes
 
-- When used with the `--lint` parameter, the `bundle` command now returns a non-zero code when it detects an error.
+- Fixed the the `bundle` command to return a non-zero code when it detects an error when used with the `--lint` option.
 
 ----
 
 ## 1.0.0-beta.64 (2021-10-20)
 
 ### Fixes
 
-- Resolved an issue with the `--format` option not working with the `bundle` command.
+- Fixed an issue with the `--format` option not working with the `bundle` command.
 
-- Resolved a validation issue with the non-string `openapi` value in API definitions. The `lint` command now warns if the value is not string instead of crashing.
+- Fixed a validation issue with the non-string `openapi` value in API definitions. The `lint` command now warns if the value is not string instead of crashing.
```

Comment:
```suggestion
- Fixed a validation issue with the non-string `openapi` value in API definitions. The `lint` command warns if the value is not string instead of crashing.
```